ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/create_table.md

```diff
@@ -159,6 +159,28 @@ WITH (
 );
 ```
 
+Creating a table with enabled [TTL](../../../../concepts/ttl.md) for automatically removing outdated records from the table. A record is deleted when the value of one of the columns (configurable by the user) of type: `Date`, `Datetime`, or `Timestamp` becomes greater than the current time (Epoch time). The lifespan of records can be extended (globally for the entire table) by the value specified in the `Interval` function:
+```sql
+CREATE TABLE my_table (
+    id Uint64,
+    title Utf8,
+    expire_at Timestamp
+    PRIMARY KEY (id)
+)
+WITH (
+    TTL = Interval("PT0S") ON expire_at
+);
+```
+
+In the provided example, a record will be deleted as soon as the value in the `expire_at` column matches the Epoch time, since the Interval function specifies the value "PT0S" is 0 seconds.
+
+{% note info %}
+
+In the table, there can be only one TTL column. If in the TTL column has a `NULL` value, that row will never be deleted.
```

Comment:
In this patch, it looks like "rows" and "records" are used as synonyms. Let's stick to one of them to minimize the risk of confusion.